domains/lumix.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "o-lumix",
+           "email": "lumixofficiel@gmail.com",
+           "discord": "828313756827516949"
+        },
+    
+        "record": {
+            "CNAME": "o-lumix.github.io"
```

Comment:
404 page, but if you have the website hosted but not on the root domain then send the website link to the page

Reply:
Now is good?